raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,332 @@
+            - assert: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+      - serial:
+          name:
+          tasks:
+            # stop node B
+            - stop_node: 1
+            # give it time to shutdown
+            - wait: 10
+      - serial:
+          name: "Multi hop payment 2"
+          tasks:
+            # 0 sends 10 tokens to 2 (using the 0 <-> 4 <-> 2 route)
+            - transfer: {from: 0, to: 2, amount: 10_000_000_000_000_000, lock_timeout: 30}
+            - wait: 5
+      - parallel:
+          name: "Assert transfer was executed correctly"
+          tasks:
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 0, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000, balance: 40_000_000_000_000_000, state: "opened"}
+            - assert: {from: 4, to: 0, total_deposit: 25_000_000_000_000_000, balance: 25_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 4, total_deposit: 10_000_000_000_000_000, balance: 10_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 4, total_deposit: 150_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 4, to: 2, total_deposit: 0, balance: 50_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000, balance: 40_000_000_000_000_000, state: "opened"}
+      - parallel:
+          name: "Check edge cases again"
+          tasks:
+            # try to open a channel with an offline node, but the channel already exists
+            - open_channel: {from: 0, to: 1, total_deposit: 200_000_000_000_000_000, expected_http_status: 409}
+            # Try to make a payment to an offline node
+            - transfer: {from: 0, to: 1, amount: 1_000_000_000_000_000, expected_http_status: 409, lock_timeout: 30}
+      - serial:
+          name: "Restarting node 1"
+          tasks:
+            - start_node: 1
+      - parallel:
+          name: "Checking for the 3 <-> 2 channel state"
+          tasks:
+            - assert: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000, balance: 40_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 160_000_000_000_000_000, state: "opened"}
+      - transfer: {from: 3, to: 2, amount: 99_000_000_000_000_000, lock_timeout: 30}
```

Comment:
This transfer seems to be a bit dislocated. Can you please add it in a `tasks` section and explain what it does?